renovate.json5

```diff
@@ -12,8 +12,6 @@
     "react-dom-17",
     "@testing-library/react-12",
     "react-test-renderer-17",
```

Comment:
Gonna be doing a follow up PR to clean up some of this old react 16 & 17 stuff

Reply:
I think @gyfchong already has some sort of a clean up branch so you can probably just tell him about it :)
https://github.com/cultureamp/kaizen-design-system/pull/4021